website/docs/d/maps_account.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
can we also add this to the sidebar? https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L119